Puppetfile

```diff
@@ -1,12 +1,9 @@
 forge 'https://forgeapi.puppetlabs.com'
 
-mod 'katello/candlepin',             :git => 'https://github.com/theforeman/puppet-candlepin'
-mod 'katello/foreman_proxy_content', :git => 'https://github.com/theforeman/puppet-foreman_proxy_content'
-mod 'katello/certs',                 :git => 'https://github.com/theforeman/puppet-certs'
-mod 'katello/katello',               :git => 'https://github.com/theforeman/puppet-katello'
-mod 'katello/pulp',                  :git => 'https://github.com/theforeman/puppet-pulp'
-mod 'katello/qpid',                  :git => 'https://github.com/theforeman/puppet-qpid'
-
-# A fork that marks metadata as compatible with puppet 5
-# https://github.com/thias/puppet-squid3/pull/59
-mod 'thias-squid3',                  :git => 'https://github.com/theforeman/puppet-squid3', :ref => 'allow-puppet-5'
+mod 'katello/candlepin', '>= 5.1.1 < 5.2.0'
```

Comment:
In Foreman I've kept the alignment. Perhaps do it here too? I've recently thought of how I could actually align it automatically but till that time ... :)

Reply:
Heh, I just used the `pin_modules` task so I guess I can go edit the output after :)